integration_tests/backend/README.md

```diff
@@ -0,0 +1,18 @@
+# Backend Integration Tests
+
+These tests run the Aqueduct backend.
```

Comment:
Can you write a quick sentence or two about what kinds of tests are here? And a bit about how tests are setup and cleaned up?